Automattic/wp-calypso
# Add Error Notices to Google My Business Charts

## Specs

<img width="1068" alt="screen shot 2018-05-15 at 3 05 07 pm" src="https://user-images.githubusercontent.com/2810519/40086005-6301db86-5851-11e8-93b2-6f385ae3c2ce.png">

Adds notices to charts where an error has occurred while trying to fetch the data.

## Testing

1. Introduce errors to Google My Business APIs ( see patch in D12650 )
2. Navigate to the Google My Business on an eligilbe Stats Page
3. Open browser developer console network tab
4. If necessary, change the interval of the charts until an error notice appears
5. Verify that the responses that have an "error" in the body show an error notice on the chart

## Review
